.gitignore

```diff
@@ -1,3 +1,4 @@
+downloaded/specs
```

Comment:
This shouldn't be here now that we have the `--fetch-specs` flag per our pervious conversation.